cmd/influx/authorization.go

```diff
@@ -226,15 +218,15 @@ func authorizationCreateF(cmd *cobra.Command, args []string) error {
 
 	authorization := &platform.Authorization{
 		Permissions: permissions,
-		OrgID:       o.ID,
+		OrgID:       orgID,
 	}
 
 	if userName := authCreateFlags.user; userName != "" {
 		userSvc, err := newUserService()
 		if err != nil {
 			return err
 		}
-		user, err := userSvc.FindUser(ctx, platform.UserFilter{
+		user, err := userSvc.FindUser(context.Background(), platform.UserFilter{
```

Comment:
i think we would like to hook up the context sometime in the future, that's why the ctx is defined on top

Reply:
I wish that were the case, but the context that was removed above waws just `context.Background()` 🤷‍♂. It just happened to be instantiated well before its use case.